plugin-server/tests/shared/db.test.ts

```diff
@@ -59,6 +62,93 @@ describe('DB', () => {
         })
     })
 
+    async function fetchPersonByPersonId(teamId: number, personId: number): Promise<Person> {
+        const selectResult = await db.postgresQuery(
+            `SELECT * FROM posthog_person WHERE team_id = $1 AND id = $2`,
+            [teamId, personId],
+            'fetchPersonByPersonId'
+        )
+
+        return selectResult.rows[0]
+    }
+
+    describe('createPerson', () => {
+        let team: Team
+        let person: Person
+        const uuid = new UUIDT().toString()
+        const distinctId = 'distinct_id1'
+
+        const TIMESTAMP = DateTime.fromISO('2000-10-14T11:42:06.502Z')
+
+        beforeEach(async () => {
+            team = await getFirstTeam(hub)
+        })
+
+        test('witout properties', async () => {
+            const person = await db.createPerson(TIMESTAMP, {}, {}, team.id, null, false, uuid, [distinctId])
+            const fetched_person = await fetchPersonByPersonId(team.id, person.id)
+
+            expect(fetched_person.is_identified).toEqual(false)
+            expect(fetched_person.properties).toEqual({})
+            expect(fetched_person.properties_last_operation).toEqual({})
+            expect(fetched_person.properties_last_updated_at).toEqual({})
+            expect(fetched_person.uuid).toEqual(uuid)
+            expect(fetched_person.team_id).toEqual(team.id)
+        })
+
+        test('witout properties indentified true', async () => {
+            const person = await db.createPerson(TIMESTAMP, {}, {}, team.id, null, true, uuid, [distinctId])
+            const fetched_person = await fetchPersonByPersonId(team.id, person.id)
+            expect(fetched_person.is_identified).toEqual(true)
+            expect(fetched_person.properties).toEqual({})
+            expect(fetched_person.properties_last_operation).toEqual({})
+            expect(fetched_person.properties_last_updated_at).toEqual({})
+            expect(fetched_person.uuid).toEqual(uuid)
+            expect(fetched_person.team_id).toEqual(team.id)
+        })
+
+        test('with properties', async () => {
```

Comment:
can we assert here that `updatePerson` was never called?

Writing from memory:

```js
jest.spyOn(hub.db, 'updatePerson')
expect(updatePerson).not.toHaveBeenCalled()
```

Reply:
btw this needed to be in process-event.ts to actually test anything as create never calls update